source/overview/compliance.rst

```diff
@@ -1,6 +1,76 @@
+
+- **Security Infrastructure** - Continual investment in security, privacy and compliance capabilities 
+- **Contractual Obligations** - Appropriate contractual obligations through our terms of service, including the Data Processing Addendum in our standard **Terms of Service**
+- **Privacy Measures** - Self-certification to international standards for data transfer standards including **E.U.-U.S. Privacy Shield** 
+- **Product Features** to ensure data management and data portability
+
+To say up to date with our efforts over time, please refer back to this site or subscribe to `our regular newsletter <https://about.mattermost.com/newsletter/>`_. 
+
+Security Infrastructure 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost enables organizations to protect their information, and the information of their users and customers, through self-hosted communication infrastructure developed with a high standard of security, including: 
+
+- **Security Features** in Mattermost open source and commercial offerings that enable deployment on your organization’s own infrastructure
+- **Responsible Disclosure Policy** for security researchers around the world to confidentially report suspected vulnerabilities, which can be addressed in updates to Mattermost software 
+- **Security Reviews** conducted by both our own internal security review team and external security researchers
+- **ISO 27001 Standards** which are met to achieve alignment with international security guidelines
+
+Contractual Obligations 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost adheres to contractual obligations for ensuring the proper management of data through: 
+
+- **GDPR compliant Data Processing Addendum** included with Mattermost’s standard terms
+- **Mattermost Privacy Policy** sharing how data is handled on the online infrastructure controlled by Mattermost, Inc. 
+
+Privacy Measures 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost self-certifies that it adheres to the **E.U.-U.S. Privacy Shield** principles of notice, choice, onward transfer, security, data integrity, access and enforcement for personal data submitted by our customers and partners.
+
+
+Product Features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost supports features that ensure data management and data portability.
+
+Data Management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Data retention**: Use `data retention <https://docs.mattermost.com/administration/data-retention.html>`_ to automatically erase data after a set period of time, meeting Right to Erasure principle. In Team Edition, you can use database scripts.
+
+- **Profile deletion**: Delete a person’s personal information via `a command line tool <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-delete>`_. The command permanently deletes all user information including message contents and files made by the user.
```

Comment:
Not true. User's files uploaded are not deleted.

Reply:
Will update